run-local.sh

```
@@ -0,0 +1,76 @@
+# Set up Python environment
+if [ ! -d ".venv" ]; then
+    echo "Creating Python virtual environment..."
+    python -m venv .venv
+fi
+
+# Activate virtual environment
+source .venv/bin/activate || source .venv/Scripts/activate
+
+# Install Python requirements
+echo "Installing Python requirements..."
+cd api
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)